### PR TITLE
chore: fix some comments 

### PR DESCRIPTION
**Pull request by findmyhappy**
# Objective

remove repetitive words

### CODE REVIEW

File: crates/bevy_pbr/src/render/parallax_mapping.wgsl [hunk 1]
```diff
@@ -56,7 +56,7 @@ fn parallaxed_uv(
     var texture_depth = sample_depth_map(uv);
 
     // texture_depth > current_layer_depth means the depth map depth is deeper
-    // than the depth the ray would be at at this UV offset so the ray has not
+    // than the depth the ray would be at this UV offset so the ray has not
```
Comment (atlv24) [inline crates/bevy_pbr/src/render/parallax_mapping.wgsl:59]: revert this change, it was correct before

File: crates/bevy_pbr/src/light/mod.rs [hunk 2]
```diff
@@ -1320,7 +1320,7 @@ pub(crate) fn assign_lights_to_clusters(
                 // scale x and y cluster count to be able to fit all our indices
 
                 // we take the ratio of the actual indices over the index estimate.
-                // this not not guaranteed to be small enough due to overlapped tiles, but
+                // this not guaranteed to be small enough due to overlapped tiles, but
```
Comment (atlv24) [inline crates/bevy_pbr/src/light/mod.rs:1323]:
```suggestion
                // this is not guaranteed to be small enough due to overlapped tiles, but
```

**Review by atlv24 — APPROVED** [review]

File: examples/app/logs.rs [hunk 3]
```diff
@@ -69,7 +69,7 @@ fn log_once_system() {
         info_once!("logs once per call site, so this works just fine: {}", i);
     }
 
-    // you can also use the 'once!' macro directly, in situations you want do do
+    // you can also use the 'once!' macro directly, in situations you want do
     // something expensive only once within the context of a continuous system.
```
Comment (soqb) [inline examples/app/logs.rs:73]: missing a "where" i think.

might as well also fix grammar/formatting here.

```suggestion
    // you can also use the `once!` macro directly,
    // in situations where you want to do something expensive only once
    // within the context of a continuous system.
```

**Review by soqb — APPROVED** [review]
LGTM except for one grammar issue.

**Review by james7132 — APPROVED** [review]
